### ISSUE TITLE
Paper maps from adminworld/flatlands clear after reset of main world

### ISSUE THREAD

**Issue by adamvandyck** [TotalFreedom/TotalFreedomMod#1623]
Any paper map thats made in either flatlands or adminworld gets cleared when the main map gets reset. (See the attached picture) A solution to this would be to have TFM store the map data from adminworld / flatlands in the data folder of the world that it was created in like how map data is normally stored (Example: Map data from a map created in adminworld would be stored in the data folder for adminworld). This will prevent maps from other worlds being cleared when the main map gets restored.
![2016-04-21_17 07 01](https://cloud.githubusercontent.com/assets/17771884/14727041/c9050d48-07ec-11e6-931a-6597d088e6e2.png)

**Comment by Wild1145** [comment]
I do not believe TFM handles any of this at all, so we would not be able to save it in adminworld?

**Comment by adamvandyck** [comment]
The reason why I believe this is a problem with TFM is because in normal worlds not generated by TFM (Woosh as an example), maps that are created in a world have its map data file stored in the data folder of that same world, but in the worlds that TFM generates (flatlands, adminworld) the map data files for maps created in those worlds are being stored in the data folder of the main world (Woosh I believe).

Another option could be adding a plugin called ImageOnMaps

**Comment by Wild1145** [comment]
Its possible that the way the maps are being store is incorrect, however I am not sure if this is something we can set within TFM or if it has to be done within MC its self? @JeromSar
